src/contracts/Market.ts

```diff
@@ -382,7 +407,7 @@ export const basicRemoveLiquidity = async (
   slippage: number,
 ): Promise<string> => {
   const executionDeadLine = getExecutionDeadline();
-  const exitOp = await marketContract.methods.removeLiquidity(
+  const exitOp = marketContract.methods.removeLiquidity(
```

Comment:
why is await removed only from this specific method?

Reply:
'cause we do not need to await there. we need to only await when we call `send`